MapboxDirections/Attributes/MBSpeedLimit.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+
+/**
+ A localized limit for measuring speed limits.
+ */
+@objc(MBSpeedLimit)
+public class SpeedLimit: NSObject, NSSecureCoding {
+    
+    /**
+     A unitless measure of speed which is dependent on the `MaximumSpeedLimit.speedUnits`.
+     
+     If a speed is not known, it will default to `Int.NSNotFound`.
+     */
+    @objc public var speed: Int = NSNotFound
```

Comment:
Speed limits can be non-integral, so `Float` or `Double` would be a more appropriate type. `NSNotFound` is only appropriate as a magic value for integral types, so use −1 instead.